scripting/include/srccoop/manager.inc

```diff
@@ -66,33 +66,46 @@ methodmap ChangeLevelManager < Array_t
 	public void UpdateStatus(int iSecondsLeft)
 	{
 		char szMapName[MAX_MAPNAME], szDisplay[128];
-		Panel panel = new Panel();
-		
-		Format(szDisplay, sizeof(szDisplay), "Changing map.. (%d)", iSecondsLeft);
-		panel.SetTitle(szDisplay);
-
-		ChangeLevelManager pSorted = this.Sorted();
-		ChangeLevelVoteLump pChangeLvlLump;
-		
-		for (int i = 0; i < pSorted.Length; i++)
+		if (g_pConvarEndWaitDisplayMode.BoolValue)
 		{
-			pSorted.GetArray(i, pChangeLvlLump);
-			if(pChangeLvlLump.m_pChangeLevel.IsValid())
+			Format(szDisplay, sizeof(szDisplay), "Changing map\n(%d)", iSecondsLeft);
+			SetHudTextParams(-1.0, 0.05, 1.0, 200, 200, 200, 255, 0, 0.5, 0.1, 0.1);
+			for (int i = 1; i <= MaxClients; i++)
 			{
-		 		pChangeLvlLump.m_pChangeLevel.GetMapName(szMapName, sizeof(szMapName));
-				Format(szDisplay, sizeof(szDisplay), "%s (%d players)", szMapName, pChangeLvlLump.votes);
-				panel.DrawItem(szDisplay);
+				if (IsClientInGame(i))
+					ShowHudText(i, 2, "%s",szDisplay);
```

Comment:
I have seen escape character issues before if you pass the buffer directly. But in this case, there probably shouldn't ever be any escape characters in it so it can be changed.